.github/workflows/ci.yml

```diff
@@ -49,6 +49,30 @@ jobs:
       - name: Test
         run: ./ci/${{ matrix.crates }}.sh
 
+  # Test crates on non x86_64 architectures.
+  cross:
+    strategy:
+      matrix:
+        crates:
+          - crossbeam
+          - crossbeam-channel
+          - crossbeam-deque
+          - crossbeam-epoch
+          - crossbeam-queue
+          - crossbeam-skiplist
+          - crossbeam-utils
+        target:
+          - i686-unknown-linux-gnu
```

Comment:
I'm planning to submit a patch for testing on aarch64-unknown-linux-gnu once this patch is merged. Because aarch64-unknown-linux-gnu is a target [recently promoted to tier-1](https://github.com/rust-lang/rfcs/pull/2959), and aarch (ARM architecture) has a memory model different than x86.

Reply:
Personally I'm fine with both splitting and merging the changes.

Reply:
I found that it works without ignoring channel tests, so I'll merge it into this PR.

Reply:
Done in 4ba3e3838fc74b08d24fdd30a4bf0473e15283bc